axon-server/administration/multi-context.md

```diff
@@ -2,17 +2,138 @@
+* _**-n**_ refers to the comma separated list of node names that should be members of the new context. This parameter registers them as "PRIMARY" member nodes of that context
+
+_Optional parameters_
+
+* _**-S**_ if not supplied connects by default to http://localhost:8024. If supplied, it should be any node serving the _\_admin_ context 
+* _**-a**_ refers to the comma separated list of node names that should be "ACTIVE\_BACKUP" member nodes of that context
+* _**-m**_ refers to the comma separated list of node names that should be "MESSAGING\_ONLY" member nodes of that context
+* _**-p**_ refers to the comma separated list of node names that should be "PASSIVE\_BACKUP" member nodes of that context
+* _**-t**_  refers to the access token to authenticate at server
+
+
+
+_**Deleting context\(s\)**_
+
+The delete-context command helps in the deletion of a context and its associated data from all member nodes of that context.  A sample of the command with the mandatory parameters is depicted below
+
+```
+$ java -jar ./axonserver-cli.jar delete-context -S http://[node]:[port] -c [context-name]
+```
+
+_Mandatory parameters_
+
+* _**-c**_ refers to the context that needs to be deleted
+
+_Optional parameters_
+
+* _**-S**_ if not supplied connects by default to http://localhost:8024. If supplied, it should be any node serving the _\_admin_ context 
+* _**-t**_  refers to the access token to authenticate at server
+
+
+
+_**Adding a node to a Context**_
+
+The add-node-to-context command helps in the registration of a new member node creation of an existing context. A sample of the command with the mandatory parameters is depicted below
 
-```text
-$ java -jar ./axonserver-cli.jar create-context -S http://[node]:[port] -c [context-name] -n [members]
 ```
+$ java -jar ./axonserver-cli.jar add-node-to-context -S http://[node]:[port] -c [context-name] -r [role of the node] -n [node name]‌
+```
+
+_Mandatory parameters_
+
+* _**-c**_ refers to an existing context
+* _**-n**_ refers to the node name that should be a member of this context
+* _**-r**_ refers to the role of this node within the context 
+
+_Optional parameters_
+
+* _**-S**_ if not supplied connects by default to http://localhost:8024. If supplied, it should be any node serving the _\_admin_ context 
+* _**-t**_  refers to the access token to authenticate at server
+
+
+
+_**Deleting a node from a context**_
+
+The delete-node-from-context command helps in the deletion member node from an existing context. A sample of the command with the mandatory parameters is depicted below
+
+```
+$ java -jar ./axonserver-cli.jar delete-node-from-context -S http://[node]:[port] -c [context-name] -n [node name]‌
+```
+
+_Mandatory parameters_
+
+* _**-c**_ refers to an existing context
+* _**-n**_ refers to the node name that should be a member of this context
```

Comment:
Wouldn't it make more sense to adjust this sentence to this:
`* _**-n**_ refers to the node name that should _no longer_ be a member of this context`
?